packages/web3-core/src/web3_context.ts

```diff
@@ -19,6 +19,8 @@ export class Web3Context<API extends Web3APISpec> extends Web3Config {
 		return this.requestManager.provider;
 	}
 
+	// TODO Getting: warning  Unexpected any. Specify a different type
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
This randomly started throwing, not sure what to do about it @nazarhussain

Reply:
Let's proceed with it. Eventually we have to rewise all `eslint-disable-next-line` in the source code after all.